### PR TITLE
Upgrade to OpenEthereum v3.2.5

### PR DESCRIPTION
**Pull request by yucer-elbt**
According to configuration changes proposed in [EWF
Docs](https://energyweb.atlassian.net/wiki/spaces/EWF/pages/706478224/Updating+the+Client).

### CODE REVIEW

**Review by royki — APPROVED** [review]

**Comment by yucer-elbt** [comment]
I just noted that the initial status of `disable_periodic` in the (instructions)[https://energyweb.atlassian.net/wiki/spaces/EWF/pages/706478224/Updating+the+Client] was:

`disable_periodic = true`

The setting was removed... and `enable=false` remains in its place with:

`enable = false`

but here in the *ewf-rpc* the setting was:

`disable_periodic = false`  

should the value of enable be `true` ?:

`enable = true` 

I mean is the new setting related with the old one ? Or are they just distinct settings?

**Comment by royki** [comment]
> I just noted that the initial status of `disable_periodic` in the (instructions)[https://energyweb.atlassian.net/wiki/spaces/EWF/pages/706478224/Updating+the+Client] was:
> 
> `disable_periodic = true`
> 
> The setting was removed... and `enable=false` remains in its place with:
> 
> `enable = false`
> 
> but here in the _ewf-rpc_ the setting was:
> 
> `disable_periodic = false`
> 
> should the value of enable be `true` ?:
> 
> `enable = true`
> 
> I mean is the new setting related with the old one ? Or are they just distinct settings?

For Validator node, there is no need to have snapshot. In OpenEthereum , this is by default enabled now. That's why we made it false. And previous setup also it was disable via `disable_periodic`

For RPC snapshot is enabled. 
Sorry I didn't notice this. Good catch.
Please update this.

File: volta-rpc/install-rpc-ubuntu-server-18.04-volta.bash [hunk 1]
```diff
@@ -205,14 +200,13 @@ warp = false
...
 max_pending_peers = 64
-no_serve_light = false
 reserved_peers = "/parity/config/peers"
 
 [footprint]
 db_compaction = "ssd"
 
 [snapshots]
-disable_periodic = false
+enable = false
```
Comment (royki) [inline volta-rpc/install-rpc-ubuntu-server-18.04-volta.bash]: It should be `enable = true`

File: ewc-rpc/install-rpc-ubuntu-server-18.04-ewc.bash [hunk 2]
```diff
@@ -205,14 +200,13 @@ warp = false
...
 max_pending_peers = 64
-no_serve_light = false
 reserved_peers = "/parity/config/peers"
 
 [footprint]
 db_compaction = "ssd"
 
 [snapshots]
-disable_periodic = false
+enable = false
```
Comment (royki) [inline ewc-rpc/install-rpc-ubuntu-server-18.04-ewc.bash]: It should be `enable = true`

**Comment by yucer-elbt** [comment]
@royki 

I think it is fixed now.

**Review by royki — APPROVED** [review]

**Pull request merged**